repository_service_tuf_worker/repository.py

```diff
@@ -398,112 +427,172 @@ def bootstrap(
 
         return asdict(result)
 
-    def add_targets(self, payload: Dict[str, Any], update_state: str) -> None:
+    def publish_targets(self):
+        """
+        Publish targets from SQL DB as persistent TUF Metadata in the backend
+        storage.
+        """
+
+        # lock to avoid race conditions
+        with self._redis.lock("publish_targets", timeout=5.0):
+            # get all delegated role names with unpublished targets
+            unpublished_roles = targets_crud.read_unpublished_rolenames(
+                self._db
+            )
+
+            if len(unpublished_roles) == 0:
+                logging.debug("No new targets in delegated roles. Finishing")
+                return None
+
+            # initialize the new snapshot targets meta and published targets
+            # from DB SQL
+            new_snapshot_meta: List[Tuple(str, int)] = []
```

Comment:
I suggested it before in https://github.com/vmware/repository-service-tuf-worker/pull/145#discussion_r1049654442 and you resolved it, but it was somehow lost.

Reply:
You've suggested shortening the name from  `new_snapshot_targets_meta` to `new_snapshot_meta`. Not sure what I'm missign.

Reply:
You are right, my mistake.